phy/repl.nim

```diff
@@ -107,7 +107,11 @@ proc process(ctx: var ModuleCtx, reporter: Reporter,
       unreachable("memory config invalid; there's probably a bug in source2il")
 
     var env = initVm(mem.total, mem.total)
-    link(env, hostProcedures(includeTest = false), [module])
+    var ltab = LinkTable()
```

Comment:
OK, I kinda see how we could have module definition swapping, using the `LinkTable` to link, and more importantly unlink/relink, procedures if we get updated definitions.

Reply:
Yeah, that would work. Not right now, as `load` doesn't touch existing exported procedures, but it's fairly easy to support.